lib-multisrc/goda/src/eu/kanade/tachiyomi/multisrc/goda/GoDa.kt

```diff
@@ -80,11 +80,13 @@ open class GoDa(
 
     private fun Element.getMangaId() = selectFirst("#mangachapters")!!.attr("data-mid")
 
+    open val elementCheck: Int = 6
+
     override fun mangaDetailsParse(response: Response) = SManga.create().apply {
         val document = response.asJsoup().selectFirst("main")!!
         val titleElement = document.selectFirst("h1")!!
         val elements = titleElement.parent()!!.parent()!!.children()
-        check(elements.size == 6)
+        check(elements.size == elementCheck)
```

Comment:
```suggestion
        check(elements[4].tagName() == "p")
```

The element count varies across mirrors, as the linked issue's comments discovered. Let's change it to a more stable check. Remove the `elementCheck` field.

Reply:
btw, I move the check to below status since status do not use elements, prevent status from breaking when check is false

Reply:
`check()` will throw an exception. Should the page structure change, moving it below will not make the status data return successfully.

Reply:
O, good to know, I still very new to android coding. Decided to do it because when it throw "illegal State Exception" error, the status is showing unkown when it should not affect it.